<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T4zet_aoq_ranges</class>
 <widget class="QDialog" name="T4zet_aoq_ranges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textLabel1">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Many 2D spectra of FRS show Z and A / Q values.
Here you may define what is the interesting region of 
Z or A / Q in your particular experiment.
</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout" name="_2">
       <item>
        <layout class="QHBoxLayout" name="_3">
         <item>
          <widget class="QLabel" name="textLabel2">
           <property name="text">
            <string>Z  min</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_zet_min">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>78</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_4">
         <item>
          <widget class="QLabel" name="textLabel2_3">
           <property name="text">
            <string>A / Q  min</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_aoq_min"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_5">
       <item>
        <layout class="QHBoxLayout" name="_6">
         <item>
          <widget class="QLabel" name="textLabel2_2">
           <property name="text">
            <string>Z max</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_zet_max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_7">
         <item>
          <widget class="QLabel" name="textLabel2_2_2">
           <property name="text">
            <string>A / Q  max</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_aoq_max"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1_2">
     <property name="text">
      <string>These values will be used for example if you have the matrix 
Z vs A / Q. Or any other, which have one of axies Z or A / Q </string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_8">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_zet_min</tabstop>
  <tabstop>lineEdit_zet_max</tabstop>
  <tabstop>lineEdit_aoq_min</tabstop>
  <tabstop>lineEdit_aoq_max</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>T4zet_aoq_ranges</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>T4zet_aoq_ranges</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
